Theme color variables go consistently from dark to bright or vice-versa

spoenemann opened the pull request:
Fixes #1504. I tried to keep the two themes intact as much as possible, but a few changes were necessary. Please check the result.

jbicker commented on packages/core/src/browser/style/variables-dark.useable.css:
```diff
@@ -1,128 +1,115 @@
-  --theia-accent-color5: var(--md-grey-800);
+  --theia-accent-color1: #28abe7;
+  --theia-accent-color2: #2288be;
+  --theia-accent-color3: #265f8a;
```
Maybe, the accent-colors are too greyish now? 
For example in history list the contrast is very low:
<img width="308" alt="screen shot 2018-04-10 at 16 43 28" src="https://user-images.githubusercontent.com/28291421/38564328-36b4fb36-3cdf-11e8-9002-b21dd89764ed.png">

spoenemann replied:
Made the font used for commit time a bit darker, and made the accent color brighter for the bright theme.

spoenemann commented on packages/core/src/browser/style/variables-dark.useable.css:
```diff
@@ -156,40 +143,45 @@ all of MD as it is not optimized for dense, information rich UIs.
   --theia-ui-button-font-color: var(--theia-ui-font-color1);
   --theia-ui-button-color-secondary: var(--theia-accent-color5);
   --theia-ui-button-font-color-secondary: var(--theia-ui-font-color1);
   --theia-ui-button-color-disabled: var(--theia-accent-color5);
```
Switched to `--theia-accent-color3`.

spoenemann commented on packages/core/src/browser/style/menus.css:
```diff
@@ -43,14 +43,14 @@
 
 .p-MenuBar-item.p-mod-active {
-  background: var(--theia-layout-color2);
+  background: var(--theia-menu-color1);
```
Made that a bit brighter.

spoenemann commented:
@jbicker I addressed your comments, but had to change a few variables which might have other side effects. Could you check again please?

Currently it is very difficult to get the themes right, mainly because the variables are used so inconsistently throughout the application. Hopefully fixing #1486 would improve the situation.

svenefftinge commented:
Inactive editor tabs in light theme are too dark:

<img width="323" alt="screen shot 2018-04-13 at 10 45 41" src="https://user-images.githubusercontent.com/372735/38725534-eb38d4a4-3f07-11e8-996d-f1069325b84a.png">

spoenemann commented:
> Inactive editor tabs in light theme are too dark

Corrected that.

I also removed a few variables that were almost unused.
